test/testing_common.py

```diff
@@ -3,9 +3,24 @@
 import shutil
 from unittest import TestCase
 
+# testing base folder
```

Comment:
Also here, could you give me a hint why this is a better order for you? :)

Reply:
See above - `os.environ.setdefault('LNDMANAGE_HOME', lndmanage_home)` and `os.makedirs` before importing settings.